.buildkite/pipeline.yml

```diff
@@ -1,44 +1,53 @@
 steps:
-  - label: "GPU julia v1.6"
+  - label: ":julia: Julia {{matrix.julia}} + CUDA GPU"
     plugins:
       - JuliaCI/julia#v1:
-          version: "1.6"
```

Comment:
Without this, we have no way to tell if changes to NNlib break NNlibCUDA on Julia <1.9. So either we add something like I mentioned in https://github.com/FluxML/NNlib.jl/issues/495#issuecomment-1586234203 or we create a separate Reverse CI step/pipeline for NNlibCUDA.

Reply:
I don't understand the issue here. Changes to NNlib won't effect julia < 1.9 users since NNlib is julia >= 1.9 from now on

Reply:
For backports, if there is ever gonna be one, we can test locally.

Reply:
I missed that Project.toml also bumped Julia compat. To be clear then, merging this would mean we're stopping feature development for Julia <1.9 and now maintaining _two_ backport branches (one for NNlib and one for NNlibCUDA)? I recall there being mixed success with backport branches in FluxML before, are there any lessons learned from that so that this doesn't run into the same issues? cc @FluxML/committers

Reply:
> To be clear then, merging this would mean we're stopping feature development for Julia <1.9 and now maintaining two backport branches (one for NNlib and one for NNlibCUDA)?

we are stopping development for julia < 1.9 and we will maintain backport branches in case we feel the need and care enough about backporting something, which I consider an unlikely event. In expectation, the benefits far outweigh the drawbacks.

Reply:
1 vote for moving to 1.9 soon!

Reply:
Ok, in that case I'll cast my vote for this too. The rest of the PR LGTM.